### PR TITLE
Feature/privacy policy

### PR DESCRIPTION
**Pull request by Jean-005**

### CODE REVIEW

File: capstone_frontend/src/App.jsx [hunk 1]
```diff
@@ -84,8 +88,9 @@ function App() {
     <>
       <ThemeProvider theme={theme}>
         <DeliveryContainer />
+        <PrivacyPolicy /> 
       </ThemeProvider>
```
Comment (Aebel-Shajan) [inline capstone_frontend/src/App.jsx:92]: You can remove this from here. It is not needed

File: capstone_frontend/src/components/forms/Login.jsx [hunk 2]
```diff
@@ -15,6 +16,8 @@ const Login = ({ drivers, handleUserLogin, addNewDriver }) => {
         password: ''
     });
 
+    const [showPrivacyPolicy, setShowPrivacyPolicy] = useState(false);
```
Comment (Aebel-Shajan) [inline capstone_frontend/src/components/forms/Login.jsx:19]: This is also not required

**Review by sahilpatel1906 — CHANGES_REQUESTED** [review]
Do you need the state for show policy as its not a modal?

**Review by Aebel-Shajan — CHANGES_REQUESTED** [review]
Try changing it to a modal.

**Review by Tommybui28 — APPROVED** [review]
Lets implement the changes requested!

**Review by ney601 — CHANGES_REQUESTED** [review]
Could use modal for privacy policy

**Review by Aebel-Shajan — APPROVED** [review]
All good

**Pull request merged**